resources/scopes/README.md

```diff
@@ -0,0 +1,22 @@
+# Scopes and Commit Types
+
+This folder contains definitions for valid scopes and commit types for Keyman.
+
+[Scope Documentation](https://docs.google.com/document/d/10CTCXbNrMpl-W9gXM2fRUdrN9_toGvsqiXhk8S2CfEw/edit#)
```

Comment:
I copied the doc to the [wiki](https://github.com/keymanapp/keyman/wiki/Keyman-Project-Scopes) for easier access

Reply:
wonder if [pull request and commit workflow](https://github.com/keymanapp/keyman/wiki/Pull-Request-and-Commit-workflow-notes) gets updated to cross reference it.

Reply:
I've updated to point to the wiki and edited the PR and commit workflow as well.